pages/games.tsx

```diff
@@ -18,11 +17,145 @@ export default function Games({ games: initialGames }: { games: Game[] }) {
   if (error) return <p>An error has occurred.</p>;
   if (!data) return <p>Loading...</p>;
   const games = data || initialGames;
+  const gameList: any = [];
```

Comment:
We already have a custom hook for fetching games under utils/api/games called useGetGames. Use that instead of calling useQuery in line 14.